### PR TITLE
Prepare idr0051 for import

### PR DESCRIPTION
**Pull request by dominikl**
Added all the necessary files (bulk.yml, annotation.csv, etc.)

Two questions remain:
- If I recall correctly the next release won't have the klb reader included, so shoud the two klb files be excluded for now, ie. only three images should be imported?
- ~~How I can get the pattern files onto `/uod/idr/filesets/...`(read-only fs) again? @manics ? The pattern files are in the scripts folder and they'd have to go to https://github.com/dominikl/idr0051-fulton-tailbudlightsheet/blob/9018cd63a56c65298c3d2c059ed8d3c962da37b2/scripts/add_patternfiles.sh After that I could do test import on east.~~ (not necessary)

### CODE REVIEW

**Comment by manics** [comment]
https://idr.openmicroscopy.org/webclient/?show=image-3261692 https://github.com/IDR/idr-metadata/blob/master/idr0026-weigelin-immunotherapy/patterns/3.67.6-3.141020_11-39-56.01.pattern

Looks like the pattern files can be kept in idr-metadata?

**Comment by dominikl** [comment]
Hm, that's interesting... thought they have to be in the same place. I'll try that.

**Comment by dominikl** [comment]
The path in the pattern file doesn't work unfortuntaly "No imports found".

**Comment by dominikl** [comment]
Sorry, it works... issue was I had the line `readers: "readers.txt"` in the bulk config, and no `readers.txt`, then the importer couldn't read the pattern files.

**Comment by dominikl** [comment]
There are still various issues...
Four of the images have:
```
...
2018-11-08 14:07:31,979 22712      [2-thread-1] ERROR        ome.formats.importer.ImportLibrary - Error on import: A checksum mismatch has occurred.
omero.ChecksumValidationException: null
```
And one didn't even get that far:
```
...
2018-11-08 14:07:33,500 1047       [      main] ERROR     ome.formats.importer.cli.ErrorHandler - FILE_EXCEPTION: /home/omero/idr0051-fulton-tailbudlightsheet/experimentA/../patterns/embryo_dmso_2_new_17-00-44_p00_c00_reg_preview.klb.pattern
loci.formats.FormatException: File #64 (/uod/idr/filesets/idr0051-fulton-tailbudlightsheet/20181025-ftp/Dataset_1/Processed Image Files/embryo_dmso_2_new_17-00-44_p00_c00_t0064_reg_preview.klb) does not exist.
```

And indeed there are a couple of files missing (timepoint 64 to 79):
```
...
embryo_dmso_2_new_17-00-44_p00_c00_t0063_reg_preview.klb  
embryo_dmso_2_new_17-00-44_p00_c00_t0080_reg_preview.klb
...
```

**Comment by dominikl** [comment]
No idea where the 'ChecksumValidationException: null' comes from, Blitz.log looks ok. I downloaded  one image set, and imported it to eel (with default upload file transfer, no ln-s obviously), worked fine, too. Any ideas (/cc @sbesson ) ?
RE missing files: Shall we contact the submitter again (/cc @francesw )? That seemed to have happened already a couple of times in the past apparently...

**Comment by dominikl** [comment]
The two klb files are not included in the filepath.tsv yet. To include them depends if the release idr0051 is targeted for will have the klb reader or not.

File: experimentA/idr0051-experimentA-assays.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Dataset Name	Image File	Characteristics [Organism]	Term Source 1 REF	Term Source 1 Accession	Characteristics [Organism Part]	Term Source 2 REF	Term Source 2 Accession	Characteristics [Ecotype]	Term Source 3 REF	Term Source 3 Accession	Characteristics [Genotype]	Characteristics [Developmental Stage]	Protocol REF 1	Protocol REF 2	Protocol REF 3	Channels	Image File Type 	Comment [Image File Type]	Comment [Image File]
+Sample 3	180712_H2B_22ss_Courtney1_20180712-163837_p00_c00_preview.pattern	Danio rerio	NCBITaxon	7955	Tailbud	ZFA	ZFA_0000077	T�pfel long fin	EFO	EFO_0003332	H2B-GFP	22 Somite Stage	Lightsheet Protocol	Online tracking light sheet timelapse protocol	TGMM tracking	GFP	TIFF	preview, registered	Time point 0 - 78
```
Comment (sbesson) [inline experimentA/idr0051-experimentA-assays.txt:2]: This file contains a few symbols like `T�pfel` - does this work correctly at annotation time?

**Review by sbesson — COMMENTED** [review]
Looks good. Has the last commit with the updated timestamped folder for `Dataset1` been tested at import time?

File: experimentA/idr0051-experimentA-bulk.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+continue: true
+transfer: "ln_s"
+exclude: "clientpath"
+skip: "all"
+output: "yaml"
```
Comment (sbesson) [inline experimentA/idr0051-experimentA-bulk.yml:7]: As these are multi-file filesets aggregated via pattern files, I suspect setting `parallel_upload` to a value around 102-20 might have some impact on import speed.

File: experimentA/idr0051-experimentA-assays.txt (hunk 1 above)
Comment (dominikl) [reply experimentA/idr0051-experimentA-assays.txt:2]: I just manually replaced that with 'u' in the annotation.csv.

**Comment by dominikl** [comment]
Yes, idr0051 on idr-testing has been imported with the updated/reuploaded Dataset1 files.

**Pull request merged**